ext/drreg/drreg.c

```diff
@@ -65,17 +65,49 @@
 
 #define PRE instrlist_meta_preinsert
 
-/* This should be pretty hard to exceed as there aren't this many GPRs */
+/* We add additional spill slots to cater for client spills.
+ * The number of spills should be pretty hard to exceed as there aren't
+ * that many GPRs.
+ */
 #define MAX_SPILLS (SPILL_SLOT_MAX + 8)
+/* We choose the number of available slots for spilling simds to match their
+ * theoretical max number on any platform, which is up to 32 for 64-bit on x86.
+ */
+#define MAX_SIMD_SPILLS DR_NUM_SIMD_VECTOR_REGS
 
-#define AFLAGS_SLOT 0 /* always */
+#ifdef X86
+#    define XMM_REG_SIZE 16
+#    define YMM_REG_SIZE 32
+#    define ZMM_REG_SIZE 64
+#    define REG_SIMD_SIZE ZMM_REG_SIZE
```

Comment:
nit: in test detach_state test we called it SIMD_REG_SIZE, maybe do the same here

Reply:
Done.